examples/sklearn/mpg-estimation/predictor.py

```diff
@@ -1,13 +1,8 @@
-from joblib import load
+import mlflow.sklearn
 import numpy as np
 
 
-model = None
-
-
-def init(model_path, metadata):
```

Comment:
This was an example to show how global variables can be shared across functions using the `global` keyword. We should consider leaving it to showcase different ways.